src/proof/alethe/alethe_post_processor.cpp

```diff
@@ -60,6 +60,177 @@ bool AletheProofPostprocessCallback::update(Node res,
     {
       return addAletheStep(AletheRule::ASSUME, res, res, children, {}, *cdp);
     }
+    // See proof_rule.h for documentation on the SCOPE rule. This comment uses
+    // variable names as introduced there. Instead of (not (and F1 ... Fn))
+    // (cl (not (and F1 ... Fn))) is printed and instead of (=> (and F1 ... Fn)
+    // F) the term (cl (=> (and F1 ... Fn))) is printed while the proof node
+    // remains the same.
+    //
+    // Let (not (and F1 ... Fn))^i denote the repetition of (not (and F1 ...
+    // Fn)) for i times.
+    //
+    // T1:
+    //
+    //   P
+    // ----- ANCHOR    ------- ... ------- AND_POS
+    //  VP1             VP2_1  ...  VP2_n
```

Comment:
A question: how are you handling the fact that you'd need to introduce an anchor step marking the beginning of the subproof? Is this handled in the printer?

Reply:
Yes, in src/proof/alethe/alethe_proof_rule.h the ANCHOR_SUBPROOF rule is added as a special rule. This indicates to the printer that it should print a subproof.